hello-spring/descriptors/ModuleDescriptor-template.json

```diff
@@ -45,11 +45,11 @@
     },
     "env": [
       { "name": "JAVA_OPTIONS", "value": "-XX:MaxRAMPercentage=66.0" },
-      { "name": "DB_HOST", "value": "postgres" },
+      { "name": "DB_HOST", "value": "localhost" },
       { "name": "DB_PORT", "value": "5432" },
-      { "name": "DB_USERNAME", "value": "folio_admin" },
-      { "name": "DB_PASSWORD", "value": "folio_admin" },
-      { "name": "DB_DATABASE", "value": "okapi_modules" },
+      { "name": "DB_USERNAME", "value": "postgres" },
+      { "name": "DB_PASSWORD", "value": "nhan3004" },
```

Comment:
i really hope you don't use this password for anything

Reply:
Oops, but nah that's just a quick password I used to set it up quickly.

Reply:
ok cool, just wanted to make sure! don't want you accidentally leaking something